dts/bindings/charger/nxp,pf1550-charger.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c), 2024 Arduino SA
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP PF1550 battery charger
+
+include: battery.yaml
+
+compatible: "nxp,pf1550-charger"
+
+properties:
+  constant-charge-current-max-microamp:
+    required: true
+    type: int
+
+  vbus-current-limit-microamp:
+    type: int
+    required: true
+
+  battery-termination-microvolt:
```

Comment:
Looking at the datasheet this is the charge regulation voltage. The `const-charge-voltage-max-microvolt` property should be used.